tests/lib/cli-engine.js

```diff
@@ -1457,12 +1535,13 @@ describe("CLIEngine", function() {
 
                 engine.executeOnFiles([file]);
 
-                assert.isTrue(fs.existsSync(path.resolve(".eslintcache")), "the cache for eslint was created");
+                assert.isTrue(fs.existsSync(cacheFile), "the cache for eslint was created");
 
                 engine = new CLIEngine({
                     useEslintrc: false,
                     // specifying cache true the cache will be created
                     cache: false,
+                    cacheFile: cacheFile,
```

Comment:
@nzakas @gyandeeps this setting was missing, so CLIEngine did not remember which file to delete. Will open an issue for that feature, because it might not be intuitive that disabling the cache still needs `cacheFile` to be set or it just deletes the file when `cache` is false.